assets/scripts/aircraft.js

```diff
@@ -714,16 +753,6 @@ var Aircraft=Fiber.extend(function() {
       else {
         this.hideStrip();
 
-        // Fly away!
-        this.fms.setCurrent({
-          navmode: "heading",
-          heading: this.radial,
-          turn: null,
-          hold: false,
-          altitude: 20000,
-          speed: 330,
-        });
-
```

Comment:
Understood. I was removing it because it would then peel the departures off of their flight plan when flying a SID, and just put them on a heading away from the airport. It looks better to just leave them alone, and allow them to continue along the SID path. Of course, it would be possible that aircraft may not always leave the game, but after adding SIDs, the only time this block of code would be needed (that I can think of now) would be if they were told to fly to a fix outside of the airspace, and then another inside the airspace.

So I actually think when SIDs are implemented, this should be removed.